lib/lhm/connection.rb

```diff
@@ -2,27 +2,36 @@
 require 'lhm/sql_retry'
 
 module Lhm
-  class Connection < SimpleDelegator
-
     # Lhm::Connection inherits from SingleDelegator. It will forward any unknown method calls to the ActiveRecord
     # connection.
-    alias connection __getobj__
-    alias connection= __setobj__
```

Comment:
Was very confusing to call Lhm.connection.connection... The new name should help a lot with clarity.